README.md

```diff
@@ -1,48 +1,117 @@
-# WIP :construction: :warning:
-We are currently working on a 2nd version of WebApiClient which will be based on Refit and will support GraphQL. Please look at the [issues](https://github.com/Xablu/Xablu.WebApiClient/issues) section for more details.
+# Xablu.WebApiClient   ![xablu logo ](/readmecontent/xablu_logo.png "Xablu")
 
-# Xablu.WebApiClient
-The Xablu WebApiClient is a C# HTTP library which aims to simplify consuming of Web API services in .NET projects.
+The Xablu WebApiClient is a C# HTTP library which aims to simplify consuming of Web API services in .NET projects. 
 
-## Setup
-The Xablu.WebApiClient is written according to the Xamarin Plugin philosophy. Meaning you can simply add the Xablu.WebApiClient package through 
-[NuGet](https://www.nuget.org/packages/Xablu.WebApiClient). Install the NuGet package into your shared .NET Standard project and ALL Client projects.
+> :construction: :warning: Already using this library? 
+> We have been working on a 2nd version of WebApiClient which is based on Refit and will support GraphQL. This version has may new features. Beware, upgrading from version 1 to version 2 has some breaking changes since it’s not backwards compatible. 
 
-* NuGet: [Xablu.WebApiClient](https://www.nuget.org/packages/Xablu.WebApiClient)
-* `PM> Install-Package Xablu.WebApiClient`
+## Table of contents  
+1. [How Xablu.WebApiClient Works] (#howto)
+2. [Download / Install] (#downloadinstal)
+3. [Build Status] (#buildstatus) 
+4. [Key Features] (#features)
+5. [Example ] (#examples)
+6. [Contributions ] (#contributions)
+7. [Feedback ] (#feedback)
+
+## How Xablu.WebApiClient Works <a name="howto"></a>
+WebApiClient is an open source library, created and maintained by Xablu. It is currently available for .NET / Xamarin. Through experience, we discovered that any .NET client app that has resilient calls to web services, uses a combination of libraries. Therefore, we built: 
+
+* A REST client library that combines the most popular libraries like Refit, Fussilade and Poly into a single package. 
+* A library that makes it easier to work with GraphQL APIs.
+
+![webapiclient model](/readmecontent/model.png "WebApiClient Model")
+
+### Refit: 
```

Comment:
@LukasThijs I remember we yesterday discussed it would be good to have a short sentence introduction before listing the libraries and you said it was a good idea, but it's not here? Something as basic as "To make it happen, we combined the following libraries:" does the work (but please feel free to rephrase)

Reply:
yes on it